manual_user_install.sh

```diff
@@ -1,73 +1,76 @@
+: ${OVOS_TTS_REPOS:="ovos-tts-plugin-mimic3-server ovos-tts-plugin-mimic ovos-tts-plugin-piper ovos-tts-server-plugin"}
+: ${OVOS_EXTRA_REPOS:="ovos-config ovos-utils ovos-bus-client ovos-plugin-manager ovos-cli-client"}
+: ${OVOS_PHAL_REPOS:="ovos-PHAL ovos-phal-plugin-connectivity-events ovos-phal-plugin-system ovos-PHAL-plugin-ipgeo ovos-PHAL-plugin-oauth ovos-phal-plugin-dashboard ovos-phal-plugin-alsa"}
+: ${OVOS_SKILLS_REPOS:="skill-ovos-volume skill-ovos-fallback-unknown skill-ovos-stop skill-alerts skill-ovos-personal skill-ovos-naptime skill-ovos-date-time"}
+: ${OVOS_EXTRA_SKILL_REPOS:="skill-ovos-weather skill-ovos-hello-world skill-ovos-ddg skill-ovos-wolfie skill-ovos-wikipedia skill-ovos-fallback-chatgpt skill-ovos-news skill-ovos-somafm skill-ovos-youtube-music"}
+
+function get_src() {
+    mkdir -p ${OVOS_SOURCE}
+    pushd ${OVOS_SOURCE}
+    echo Cloning source to ${OVOS_SOURCE}
+    for p in $OVOS_CORE_REPOS $OVOS_DINKUM_REPOS $OVOS_PRECISE_LITE_REPOS $OVOS_STT_REPOS $OVOS_TTS_REPOS $OVOS_EXTRA_REPOS $OVOS_PHAL_REPOS; do
+	echo Cloning $p
+	[[ -d $p ]] || git clone https://github.com/OpenVoiceOS/$p
+    done
+    popd
+}
+
 function install_core (){
     echo "Installing OVOS core"
     echo
 
     # set up the Raspberry Pi
-    sudo apt install -y build-essential python3-dev python3-pip swig libssl-dev libfann-dev portaudio19-dev libpulse-dev cmake libncurses-dev pulseaudio-utils pulseaudio
-    pulseaudio -D
+    echo $sudoPW | sudo -S apt install -y build-essential python3-dev python3-pip python3-venv swig libssl-dev libfann-dev portaudio19-dev libpulse-dev cmake libncurses-dev pulseaudio-utils pulseaudio
 
-    # install ovos-core
-    pip3 install git+https://github.com/OpenVoiceOS/ovos-backend-client
-    pip3 install git+https://github.com/OpenVoiceOS/ovos-core
-    pip3 install git+https://github.com/OpenVoiceOS/ovos-audio
-    pip3 install git+https://github.com/OpenVoiceOS/ovos-ocp-audio-plugin
-    pip3 install git+https://github.com/OpenVoiceOS/ovos-messagebus
-    # padatious required to support newest ovos-core
-    # pip3 install padatious
+    # make sure pulseaudio is running
+    pulseaudio --check || pulseaudio -D
 
-    # dinkum listener
-    pip3 install git+https://github.com/OpenVoiceOS/ovos-dinkum-listener
-    pip3 install git+https://github.com/OpenVoiceOS/ovos-vad-plugin-silero
-    pip3 install git+https://github.com/OpenVoiceOS/ovos-ww-plugin-pocketsphinx
+    # padatious required to support newest ovos-core
+    # fann version must be fixed as raspbian libfann-dev is too old for latest
+    pip3 install padatious fann2==1.0.7
 
-    #Precise-lite wake-word (ww) cluster
-    pip3 install git+https://github.com/OpenVoiceOS/ovos-ww-plugin-precise
-    pip3 install git+https://github.com/OpenVoiceOS/ovos-ww-plugin-precise-lite
-    pip3 install --upgrade setuptools
+    # Precise-lite wake-word (ww) cluster
+    pip3 install --upgrade setuptools wheel pip
```

Comment:
Should this be moved uphill? As in the first thing installed and upgraded?

Reply:
yes - I just kept things in pretty much the same order initially